Mergin/configuration_dialog.py

```diff
@@ -97,9 +98,11 @@ def test_connection(self):
         try:
             mc = MerginClient(url, None, username, password, get_plugin_version())
             msg = "<font color=green> OK </font>"
-        except (URLError, ValueError):
+        except (URLError, ValueError) as e:
+            QgsApplication.messageLog().logMessage(str(e))
```

Comment:
I get this 'WARNING    unknown url type: '/v1/auth/login'' maybe would be good to add source of log for all entries, like 'WARNING Mergin plugin: unknown url type: '/v1/auth/login''